docs/modules/ROOT/pages/how-to/use-integrated-registry.adoc

```diff
@@ -0,0 +1,85 @@
+= How to use the OpenShift integrated container registry
+
+The OpenShift integrated registry of each {product} zone is exposed outside the cluster at `registry.<zone>.appuio.cloud`.
+
+== Prerequisites
```

Comment:
Unless changes were be made, a user will be admin in a namespace and thus can pull and push images. Can you please add that statement and mark the following as advanced/"in detail"?

Reply:
adjusted in the latest version